ceremonies/src/tests.rs

```diff
@@ -302,7 +302,7 @@ fn assert_error(actual: DispatchResult, expected: Error<TestRuntime>) {
 #[test]
 fn registering_participant_works() {
 	new_test_ext().execute_with(|| {
-		let cid = register_test_community::<TestRuntime>(None, 0.0, 0.0);
+		let cid = perform_bootstrapping_ceremony(None, 1);
```

Comment:
This test failed after the change because it tries to register participants besides bootstrappers.